packages/cmrjs/src/cmr-utils.js

```diff
@@ -113,15 +113,23 @@ function granulesToCmrFileObjects(granules) {
  * @param {string} cmrFile.filename - the s3 uri to the cmr xml file
  * @param {string} cmrFile.metadata - granule xml document
  * @param {Object} cmrClient - a CMR instance
+ * @param {string} revisionId - Optional CMR Revision ID
  * @returns {Object} CMR's success response which includes the concept-id
  */
-async function publishECHO10XML2CMR(cmrFile, cmrClient) {
+async function publishECHO10XML2CMR(cmrFile, cmrClient, revisionId = undefined) {
```

Comment:
`revisionId = undefined` is unnecessary

Reply:
For some reason I thought it was necessary to signify that the parameter was optional 🤔